site/community/index.md

```diff
@@ -26,19 +26,11 @@ limitations under the License.
 
 # Upcoming talks
 
-* September 8th-12th, 2019
-  [ApacheCon North America 2019](https://www.apachecon.com/acna19/)
-  (Las Vegas, Nevada)
-  * Building BigData Query Optimization with Apache Calcite – Best
-    Practices from Alibaba MaxCompute (Haisheng Yuan)
-  * Combining schema-on-read and schema-provisioning in Apache Drill
-    (Aman Sinha)
-  * Simple, Portable data pipelines with Apache Beam SQL
-    (Andrew Pilloud)
-  * One SQL to Rule Them All – a Syntactically Idiomatic Approach to
```

Comment:
Should we keep some of the talks records even though they have happened ?

Reply:
Without having slides, video, or other support material, I don't see a big benefit. If there is material that can be linked to the talk then it would be great to have it.

Reply:
+1

Feel free to add to https://calcite.apache.org/community/#more-talks. (I think we should curate the list a little; e.g. don't add the same talk if given at multiple locations; and remove really old talks.)